functions/src/index.ts

```diff
@@ -35,6 +36,7 @@ app.use((req, res, next) => {
 app.use("/api/v1", userRoutes);
 app.use("/api/v1", addonRoutes);
 app.use("/api/v1", authRoutes);
+app.use("/api/v1", hostRoutes);
```

Comment:
```suggestion
app.use("/overlay", hostRoutes);
```

overlay ipv api/v1 to create a difference between the backend and the addon hosting routes

![image](https://user-images.githubusercontent.com/42371079/168069779-f74cda56-bfe7-4bc8-8cb2-0282ccb42a95.png)